problema_1.py

```diff
@@ -0,0 +1,34 @@
+# Create a function that checks the data in a list of dictionaries for specific validation rules, like checking if certain columns are non-empty, if numbers are within a range, etc.
+
+data = [
+    {'name': 'Alice', 'age': 30, 'email': 'alice@example.com'},
+    {'name': 'Bob', 'age': '', 'email': 'bob@example.com'},  # Bob are certificat de nastere, totu bine, doar au uitat astia sa puna varsta ;)
+    {'name': '', 'age': 20, 'email': 'carol@example.com'},   # Numele e uitat intentionat ;)
+    {'name': 'Dave', 'age': 200, 'email': 'dave@example.com'} # Dave e world record pt cel mai batran om
+]
+
+validation_rules = {
+    'name': {'required': True, 'type': str},
+    'age': {'required': True, 'type': int, 'min': 0, 'max': 120},
+    'email': {'required': True, 'type': str}
+}
+
+def validate_data(data, validation_rules):
```

Comment:
Please add google docstrings and type hints

Reply:
Ok